keras_cv/metrics/coco/pycoco_utils.py

```diff
@@ -0,0 +1,231 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import copy
+
+import numpy as np
+import tensorflow as tf
+from pycocotools import coco
+from pycocotools import cocoeval
+
+METRIC_NAMES = [
+    "AP",
+    "AP50",
+    "AP75",
+    "APs",
+    "APm",
+    "APl",
+    "ARmax1",
+    "ARmax10",
+    "ARmax100",
+    "ARs",
+    "ARm",
+    "ARl",
+]
+
+
+class COCOWrapper(coco.COCO):
```

Comment:
structure wise, let's make it:
1. keras_cv/metrics/pycoco_wrapper.py
2. keras_cv/callbacks/pycoco_callback.py
3. in the future if this becomes a PyMetrics, then keras_cv/metrics/pycoco_metrics.py

And we can make sure 1 can be re-used in 2 and 3 when implementing 3.

Reply:
sgtm -- done